linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -42,6 +42,9 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .insertAfter(DtabStatsFilter.role, HttpLoggerConfig.module)
       .insertAfter(Retries.Role, http.StatusCodeStatsFilter.module)
       .insertAfter(AddrMetadataExtraction.Role, RewriteHostHeader.module)
+      // ensure the client-stack framing filter is placed below the stats filter
+      // so that any malframed responses it fails are counted as errors
+      .insertAfter(StatusCodeStatsFilter.role, FramingFilter.clientModule)
```

Comment:
I think this also should be after the RequestStats module so that these malframed responses show up as failures there and also below FailureAccrual so that malframed responses contribute to failure accrual.